Source/Adobe.Target.Client/TargetClientConfig.cs

```diff
@@ -1,38 +1,49 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
 namespace Adobe.Target.Client
 {
     using System;
     using System.Net;
+    using Adobe.Target.Delivery.Model;
     using Microsoft.Extensions.Logging;
     using Polly;
     using RestSharp;
 
     /// <summary>
     /// Target ClientConfig
     /// </summary>
```

Comment:
Any reason this was renamed?

Reply:
It was renamed for consistency as most of the public API has a Target prefix, I've opened a ticket to have this also fixed in Java SDK https://jira.corp.adobe.com/browse/TNT-39914